packages/ui/atoms/Action/Action.ts

```diff
@@ -0,0 +1,64 @@
+import { Base } from '@studiometa/js-toolkit';
+import type { BaseProps, BaseConfig } from '@studiometa/js-toolkit';
+
+export interface ActionProps extends BaseProps {
+  $options: {
+    target: string;
+    method: string;
+    selector: string;
+  };
+}
+
+/**
+ * Action class.
+ */
+export class Action<T extends BaseProps = BaseProps> extends Base<ActionProps & T> {
+  static config: BaseConfig = {
+    name: 'Action',
+    options: {
+      target: String,
+      method: String,
+      selector: String,
+    },
```

Comment:
**suggestion:** Like mentioned in #161, I think we should have an option to define which event will trigger the action